<?xml version="1.0" encoding="UTF-8"?><testrun name="GenderFemaleLessTest" project="EducationFemale" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="com.revature.test.GenderFemaleLessTest" time="0.717">
    <testcase name="testMapper" classname="com.revature.test.GenderFemaleLessTest" time="0.717">
      <failure>java.lang.AssertionError: 10 Error(s): (Missing expected output (Netherlands, 53.1819) at position 2., Missing expected output (Netherlands, 53.18322) at position 3., Missing expected output (Netherlands, 0.0) at position 4., Missing expected output (Netherlands, 0.0) at position 5., Missing expected output (Netherlands, 53.6965) at position 0., Missing expected output (Netherlands, 58.09252) at position 1., Received unexpected output (Netherlands, 53.69649887084961) at position 0., Received unexpected output (Netherlands, 58.09252166748047) at position 1., Received unexpected output (Netherlands, 53.18190002441406) at position 2., Received unexpected output (Netherlands, 53.18321990966797) at position 3.)
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.mrunit.TestDriver.validate(TestDriver.java:294)
	at org.apache.hadoop.mrunit.MapDriverBase.runTest(MapDriverBase.java:151)
	at org.apache.hadoop.mrunit.TestDriver.runTest(TestDriver.java:137)
	at com.revature.test.GenderFemaleLessTest.testMapper(GenderFemaleLessTest.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
  </testsuite>
</testrun>
